packages/react-core/src/components/Button/Button.tsx

```diff
@@ -67,6 +68,10 @@ export interface ButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'r
   isDanger?: boolean;
   /** Forwarded ref */
   innerRef?: React.Ref<any>;
+  /** Adds count number right of button */
+  count?: number;
```

Comment:
I might group `isRead` and `count` into an interface since they do not not make sense alone. Maybe call it `badgeCountObject`.

Reply:
Yeah - I agree they should be combined. Since `isRead` and `count` are 2/3 Badge props, consumers can add classnames to the badge if we just opt for `badgeProps` that spread into the Badge

Reply:
I agree. It struck me as off that these were independent.